content/applications/sales/subscriptions/configuration/subscription_products.rst

```diff
@@ -1,54 +1,74 @@
+
+The :guilabel:`Product Type` for the new product is automatically set to :guilabel:`Service`.
+Subscription products can be set to other types as well; however, they currently *cannot* be set to
+:guilabel:`Storable Product`.
+
+The new product also automatically has the :guilabel:`Recurring` checkbox activated. This enables
+Odoo to recognize it as a subscription product.
+
+.. image:: subscription_products/recurring-product-form.png
+   :align: center
+   :alt: The Recurring checkbox on the product form.
+
+Next, configure the :guilabel:`Time-based pricing` tab.
+
+Click :guilabel:`Add a price` to begin defining recurring prices.
+
+In the :guilabel:`Period` column, select a recurrence period (e.g. Weekly, Monthly, Yearly). In the
+:guilabel:`Price` column, type in the price for that recurrence period.
+
+To create pricing rules for specific pricelists, select a pricelist in the :guilabel:`Pricelist`
+column. This allows customers belonging to this pricelist to receive special recurring pricing for
+the subscription product.
+
+.. image:: subscription_products/pricelist-time-based-pricing.png
+   :align: center
+   :alt: Pricelists in the “Time-based pricing” tab of the product form.
+
+Add as many lines as desired to the :guilabel:`Time-based pricing` table.
 
 .. note::
-   In the Sales tab, underneath the Subscriptions section, make sure the *Subscription products*
-   option is activated. In fact, if you create a subscription product from the **Odoo Subscriptions**
-   application, this option is selected by default. However, if you create a product from another
-   application, it is not the case.
-
-   .. image:: subscription_products/subscription-products-form.png
-     :align: center
-     :alt: View of a subscription product form in Odoo Subscriptions
-
-   You can also choose the subscription templates you want to use.
-
-   .. image:: subscription_products/subscription-products-using-subscription-templates.png
-     :align: center
-     :alt: View of a subscription product form in Odoo Subscriptions
-
-.. important::
-   Be sure to check out our documentation on how to create, edit and manage your own
-   :doc:`Subscription templates <../../subscriptions/configuration/subscription_templates>`
-   before creating your own subscription products. Once created, check out our documentation on how to
-   :doc:`Create a quotation using subscription products <../../subscriptions/sales_flow/create_a_quotation>`,
-   to complete the sales flow.
-
-.. seealso::
-  - :doc:`../../subscriptions/configuration/subscription_templates`
-  - :doc:`../../subscriptions/sales_flow/create_a_quotation`
+   An existing product can be made into a subscription product simply by marking it as
+   :guilabel:`Recurring` on the product form and configuring :guilabel:`Time-based pricing`.
+
+   A product can be both a regular sales product and a subscription product at the same time.
```

Comment:
Should we expand on this a bit more? For instance, is there a specific way to configure products so that they are both regular and subscription products? Also, does configuring a product as both affect the way that Odoo handles it in certain workflows?